Signal/src/AppDelegate.m

```diff
@@ -363,6 +369,9 @@ - (void)showErrorAlertWithTitle:(NSString *)title message:(NSString *)message
     [controller addAction:[UIAlertAction actionWithTitle:NSLocalizedString(@"OK", nil)
                                                    style:UIAlertActionStyleDefault
                                                  handler:nil]];
+    [[Environment getCurrent].signalsViewController.presentedViewController presentViewController:controller
```

Comment:
We're still in the didLaunchWithOptions at this point. I think we should dispatch_async(dispatch_get_main_queue() this.